src/HealthcareApis/HealthcareApis/Az.HealthcareApis.psd1

```diff
@@ -0,0 +1,135 @@
+#
+# Module manifest for module 'Az.HealthcareApisService'
```

Comment:
@poadhika to clarify, is the name of the module `Az.HealthcareApis` or `Az.HealthcareApisService`? There are a couple of things that need to change based on the correct naming.

Reply:
@cormac, I have replied in teams